flow-typed/point/address-details.js

```diff
@@ -0,0 +1,97 @@
+// @flow
+
+export type StateElectorate = {
+  stateElectoralName?: string,
+  stateElectoralType?: string,
+  [k: string]: mixed,
+}[]
+export type CadastralParcels = {
+  propId?: string,
+  parcelId?: string[],
+  [k: string]: mixed,
+}[]
+
+export interface PointAddressResponse {
```

Comment:
Are interfaces valid flow?

Reply:
Answer is yes, for any others interested https://flow.org/en/docs/types/interfaces/